src/test/java/Homework16.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+        @Test
+    public static void RegistrationButton() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement RegistrationButton = driver.findElement(By.id("hel"));
+        RegistrationButton.click();
+        String url2 = "https://bbb.testpro.io/registration.php";
+        driver.get(url2);
+
+
+
```

Comment:
**Please delete the empty line!** «An Empty Line is a Code Smell» 

The short answer is, They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.